README-ruRU.md

```diff
@@ -3522,31 +3519,31 @@
   ```
 
 ## Даты и время
-<!--- @FIXME --->
+
```

Comment:
Please use parens in the good/bad comments instead of dashes. This is described in the contributors guide.